### PR TITLE
Paypal Integration Guide

### PR DESCRIPTION
**Pull request by sshropshire**
### Summary of changes

 - This PR updates integration docs for the `PayPalWebCheckout` module

 ### Checklist

 ~- [ ] Added a changelog entry~

- @sshropshire

### CODE REVIEW

File: docs/PayPalWebCheckout/README.md [hunk 1]
```diff
@@ -1,62 +1,57 @@
...
-keywords: 
-contentType: docs
-productStatus: current
-apiVersion: TODO
-sdkVersion: TODO
----
+# Accepting PayPal Web Checkout Payments
 
-# Pay with PayPal Custom Integration
+The PayPal Web Checkout module in the PayPal SDK enables PayPal payments in your app.
```
Comment (jaxdesmarais) [inline docs/PayPalWebCheckout/README.md]:
```suggestion
The PayPal Web Checkout module in the PayPal SDK enables PayPal payments via a webview in your app.
```

File: docs/PayPalWebCheckout/README.md [hunk 2]
```diff
@@ -1,62 +1,57 @@
...
 
 ### 1. Add the Payments SDK  to your app
 
 #### Swift Package Manager
 
-In Xcode, follow the guide to [add package dependencies to your app](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app) and enter https://github.com/paypal/iOS-SDK as the repository URL. Select the checkboxes for each specific PayPal library you wish to include in your project.
+In Xcode, add the PayPal SDK as a [package dependency](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app) to your Xcode project. Enter https://github.com/paypal/iOS-SDK as the package URL. Tick the "PayPalWebCheckout" checkbox to add the PayPal Web Checkout package to your app.
 
-In your app's source files, use the following import syntax to include PayPal's libraries:
+In your app's source code files, use the following import syntax to include the PayPal Card module:
```
Comment (jaxdesmarais) [inline docs/PayPalWebCheckout/README.md]:
```suggestion
In your app's source code files, use the following import syntax to include the PayPal Web Checkout module:
```

File: docs/PayPalWebCheckout/README.md [hunk 3]
```diff
@@ -115,18 +120,17 @@ Configure your `PayPalWebCheckoutRequest` and include the order ID generated in
 let payPalRequest = PayPalWebCheckoutRequest(orderID: "<ORDER_ID>")
 ```
 
-You can also specify the funding source for your order which are `PayPal` (default), `PayLater` and `PayPalCredit`.
-For more information go to: https://developer.paypal.com/docs/checkout/pay-later/us/
+You can also specify one of the follwing funding sources for your order: `PayPal (default)`, `PayLater` or `PayPalCredit`.
```
Comment (jaxdesmarais) [inline docs/PayPalWebCheckout/README.md]:
```suggestion
You can also specify one of the following funding sources for your order: `PayPal` (default), `PayLater` or `PayPalCredit`.
```

**Review by jaxdesmarais — APPROVED** [review]
just a couple of cleanup comments

**Review by jcnoriega — APPROVED** [review]

**Pull request merged**